config/environments/local.py

```diff
@@ -0,0 +1,60 @@
+"""Flask Local Development Environment Configuration."""
+from os import path
+
+import redis
+from config.environments.default import Config
+
+
+class LocalConfig(Config):
+    #  Application Config
+    SECRET_KEY = "dev"
+    SESSION_COOKIE_NAME = "session_cookie"
+    FLASK_ROOT = path.dirname(
+        path.dirname(path.dirname(path.realpath(__file__)))
+    )
+    FLASK_ENV = "local"
+
+    # Hostname for this service
+    AUTHENTICATOR_HOST = "http://localhost:5000"
+
+    # Azure Active Directory Config
+    AZURE_AD_CLIENT_SECRET = "nmq8Q~acUEOPWmjfvbOEQLPZy2M38yLe1PEh_cS2"
```

Comment:
Similar, if this is not a secret can you make a comment here to make that clear?

Reply:
Yep added